scripts/benchmarks/common/upsert-pr-comment.mjs

```
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs'
```

ESLint

[error] 'fs' is defined but never used.

(unused-imports/no-unused-imports)